app/models/tournament_fighter.rb

```diff
@@ -14,4 +14,14 @@ class TournamentFighter < ActiveRecord::Base
   validates_uniqueness_of :fighter_id, scope: :tournament_id
   belongs_to :fighter
   belongs_to :tournament
+
+  after_create :add_to_default_pool
+
+  def add_to_default_pool
+    tournament = Tournament.find(tournament_id)
+    default_pool = tournament.pools.find_by_name('Unassigned')
+    if default_pool
```

Comment:
I'm not sure about this check. If you're looking for 'Unassigned' by name and it's not there, you'll get an empty array, which is truthy in Ruby. You probably want `if default_pool.any?`
What if it is empty, do you want to create an 'Unassigned' pool?